helpers/award_helper.py

```diff
@@ -25,7 +25,7 @@
 AWARD_MATCHING_STRINGS = [
     (AwardType.CHAIRMANS, (["chairman"], ["hon", "finalist"])),
     (AwardType.CHAIRMANS_HONORABLE_MENTION, (["chairman", "hon", "mention"], [])),
-    (AwardType.CHAIRMANS_HONORABLE_MENTION, (["chairman", "finalist"], [])),
+    (AwardType.CHAIRMANS_FINALIST, (["chairman", "finalist"], ["hon", "mention"])),
```

Comment:
It means it doesn't match those. Probably a non-issue since we're probably not going to be re-importing old awards.